com.unity.netcode.gameobjects/Runtime/Core/NetworkBehaviour.cs

```diff
@@ -228,6 +268,13 @@ internal void __endSendClientRpc(ref FastBufferWriter bufferWriter, uint rpcMeth
 #endif
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
Curious why to add this? Does it do something important?

Reply:
The compiler will try to inline the method (if possible) which if inlined there is no impact on the stack.

Reply:
Yeah, I can see that.  I will remove that attribute.